web3 test compile failed in main

### Description

```
> Task :web3:compileTestJava FAILED
gradle/actions: Writing build results to /home/runner/_work/_temp/.gradle-actions/build-results/__run_2-1743706489118.json
[Incubating] Problems report is available at: file:///home/runner/_work/hiero-mirror-node/hiero-mirror-node/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':web3:compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/_work/hiero-mirror-node/hiero-mirror-node/hedera-mirror-web3/src/test/java/com/hedera/hapi/node/state/token/KeyTest.java:31: warning: [deprecation] RSA_3072 in KeyOneOfType has been deprecated
                                  .map(value -> new OneOf<>(Key.KeyOneOfType.RSA_3072, value))
                                                                            ^
  /home/runner/_work/hiero-mirror-node/hiero-mirror-node/hedera-mirror-web3/src/test/java/com/hedera/hapi/node/state/token/KeyTest.java:34: warning: [deprecation] ECDSA_384 in KeyOneOfType has been deprecated
                                  .map(value -> new OneOf<>(Key.KeyOneOfType.ECDSA_384, value))
                                                                            ^
  /home/runner/_work/hiero-mirror-node/hiero-mirror-node/hedera-mirror-web3/src/test/java/com/swirlds/state/spi/WritableKVStateBaseTest.java:67: warning: [deprecation] size() in ReadableKVState has been deprecated
          when(readableKVStateBase.size()).thenReturn(2L);
                                  ^
  /home/runner/_work/hiero-mirror-node/hiero-mirror-node/hedera-mirror-web3/src/test/java/com/swirlds/state/spi/WritableKVStateBaseTest.java:69: warning: [deprecation] size() in WritableKVStateBase has been deprecated
          assertThat(writableKVStateBase.size()).isEqualTo(1L);
                                        ^
  Note: Some input files use unchecked or unsafe operations.
  Note: Recompile with -Xlint:unchecked for details.
  /home/runner/_work/hiero-mirror-node/hiero-mirror-node/hedera-mirror-web3/src/test/java/com/hedera/services/utils/EntityIdUtilsTest.java:342: error: cannot find symbol
          assertNull(toTokenId(entityId));
                     ^
    symbol:   method toTokenId(EntityId)
    location: class EntityIdUtilsTest
  1 error
  4 warnings
```

### Steps to reproduce

check the description

### Additional context

_No response_

### Hedera network

_No response_

### Version

v0.128.0-SNAPSHOT

### Operating system

None

--- a/hedera-mirror-web3/src/test/java/com/hedera/services/utils/EntityIdUtilsTest.java
+++ b/hedera-mirror-web3/src/test/java/com/hedera/services/utils/EntityIdUtilsTest.java
@@ -5,6 +5,7 @@ package com.hedera.services.utils;
 import static com.hedera.mirror.common.util.DomainUtils.toEvmAddress;
 import static com.hedera.services.utils.EntityIdUtils.accountIdFromEvmAddress;
 import static com.hedera.services.utils.EntityIdUtils.contractIdFromEvmAddress;
+import static com.hedera.services.utils.EntityIdUtils.toTokenId;
 import static com.hedera.services.utils.EntityIdUtils.tokenIdFromEvmAddress;
 import static com.hedera.services.utils.IdUtils.asAccount;
 import static com.hedera.services.utils.IdUtils.asContract;
@@ -333,7 +334,7 @@ class EntityIdUtilsTest {
                 .realmNum(REALM)
                 .tokenNum(NUM)
                 .build();
-        assertEquals(expectedTokenId, EntityIdUtils.toTokenId(id));
+        assertEquals(expectedTokenId, toTokenId(id));
     }
 
     @Test
@@ -351,7 +352,7 @@ class EntityIdUtilsTest {
                 .realmNum(REALM)
                 .tokenNum(NUM)
                 .build();
-        assertEquals(expectedTokenId, EntityIdUtils.toTokenId(entityId));
+        assertEquals(expectedTokenId, toTokenId(entityId));
     }
 
     @Test

